### PR TITLE
Update about benefits styling

### PR DESCRIPTION
**Pull request by davidrapson**
New about benefits based on latest designs from Ben W.

**Large screen**
screen shot 2015-02-03 at 11 58 23

**Small screen**
screen shot 2015-02-03 at 11 58 39

### CODE REVIEW

File: frontend/assets/stylesheets/components/_packages.scss [hunk 1]
```diff
@@ -291,40 +367,37 @@
...
-            text-align: right;
-        }
+}
+.package-feature__leadin {
+    @include fs-headline(2);
+    font-weight: 500;
+    padding: rem($gs-baseline / 2) 0 rem($gs-baseline);
+
+    @include mq(tablet) {
+        border-top: 5px solid;
```
Comment (feedmypixel) [inline frontend/assets/stylesheets/components/_packages.scss]: rem for this?

Comment (davidrapson) [inline frontend/assets/stylesheets/components/_packages.scss]: Doesn't make sense for a border.

**Comment by feedmypixel** [comment]
LGTM :+1:

**Comment by davidrapson** [comment]
Cool. Really wish we had element-queries for this kind of thing though…

**Pull request merged**